src/sndfile.c

```diff
@@ -1247,7 +1247,7 @@ sf_command	(SNDFILE *sndfile, int command, void *data, int datasize)
 			return SF_FALSE ;
 
 		case SFC_GET_CUE :
-			if (datasize != sizeof (SF_CUES) || data == NULL)
+			if (datasize < (int) sizeof (uint32_t) || data == NULL)
```

Comment:
You should use the `SIGNED_SIZEOF` macro instead of `sizeof` and then casting the result to `int`.